packages/backend/src/project_environment_main_stack_creator.ts

```diff
@@ -14,7 +14,13 @@ export class ProjectEnvironmentMainStackCreator implements MainStackCreator {
   constructor(
     private readonly scope: Construct,
     private readonly backendId: BackendIdentifier
-  ) {}
+  ) {
+    process.on('message', (message) => {
+      if (message === 'amplifySynth') {
+        (scope as App).synth({ errorOnDuplicateSynth: false });
```

Comment:
Should this be in `createDefaultStack` ?

Reply:
Updated